server/neptune/workflows/terraform.go

```diff
@@ -1,13 +1,26 @@
 package workflows
 
 import (
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/activities"
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/terraform"
+	"github.com/uber-go/tally/v4"
 	"go.temporal.io/sdk/workflow"
 )
 
 // Export anything that callers need such as requests, signals, etc.
 type TerraformRequest = terraform.Request
 
+type TerraformActivities struct {
+	activities.Terraform
+}
+
+func NewTerraformActivities(scope tally.Scope) (*TerraformActivities, error) {
```

Comment:
It's not being used right now but I added it so that we have a clear scope defined for the different types activities (`terraform and deploy`) when creating the temporal server. 

https://github.com/lyft/atlantis/blob/2265b32e143c7850855fd9636f78322105892231/server/neptune/temporalworker/server.go#L101-L114

Reply:
And I moved the original app scope to where we instantiate the github client creator 
https://github.com/lyft/atlantis/blob/74cefc7c1f298b9b2dec4676e51aec4f0c7d4abc/server/neptune/workflows/internal/activities/main.go#L23-L27